Source/WebCore/crypto/CryptoAlgorithm.cpp

```diff
@@ -65,12 +65,12 @@ void CryptoAlgorithm::deriveBits(const CryptoAlgorithmParameters&, Ref<CryptoKey
     exceptionCallback(ExceptionCode::NotSupportedError);
 }
 
-void CryptoAlgorithm::importKey(CryptoKeyFormat, KeyData&&, const CryptoAlgorithmParameters&, bool, CryptoKeyUsageBitmap, KeyCallback&&, ExceptionCallback&& exceptionCallback)
+void CryptoAlgorithm::importKey(CryptoKeyFormat, KeyData&&, const CryptoAlgorithmParameters&, bool, CryptoKeyUsageBitmap, KeyCallback&&, ExceptionCallback&& exceptionCallback, bool)
```

Comment:
nit: Could we use a named enum here instead?

Maybe something like 
`enum class UseCryptoKit : bool { No, Yes };`